pages/dkp/kommander/2.2/insights/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle: DKP Insights
+title: DKP Insights
+excerpt: Use DKP Insights to monitor and tune your environment
+menuWeight: 55
+beta: false
+enterprise: false
```

Comment:
More so curious how this works since I'm new to docs. Can you remove the beta: false and enterprise: false? I'm not sure if that has any other implications but I thought those were just for the badges? If you can, it would be nice to keep it clean if you don't need these explicitly set to false.